### DISCUSSION TITLE
Uploading to the MacOS store with Transporter fails on delivery with sandbox error on the executable

### DISCUSSION THREAD

**Discussion by Yveske1966** [AvaloniaUI/Avalonia#15809] [Q&A] [answered]
The app is published with following parameters:
dotnet publish "$PROJ_DIR/$APP_NAME.OSX.csproj" -c Release -f net8.0 -r osx-arm64 --self-contained true -p:PublishSingleFile=true

The executable is signed with:
codesign --force --entitlements "$FILE_ENTITLEMENTS" --sign "$APP_SIGNING_IDENTITY" "$APP_NAME.OSX"

The app is signed as follows:
codesign --force --entitlements "$APP_ENTITLEMENTS" --sign "$APP_SIGNING_IDENTITY" "$APPL_DIR"

Both file and app entitlement contain:
    <key>com.apple.security.app-sandbox</key>
    true

Did I miss something in https://docs.avaloniaui.net/docs/distribution-publishing/macos#app-store-packaging ?

Edit: true wasn't displayed

**Comment by thevortexcloud** [comment, marked as answer]
I have no idea what the problem is, but something here might give you a hint: https://github.com/AvaloniaUI/Avalonia/discussions/15755

**Reply by Yveske1966** [reply]
Thanks, I will have a look at it shortly

**Reply by Yveske1966** [reply]
I've recreated all my certificates and permissions and I now my app is listed in the App Store.

**Reply by MikeCodesDotNET** [reply]
Care to share a link? Would love to see it!

**Reply by Yveske1966** [reply]
Nothing fancy, just a program to view downloadable pictures from performances of our dance school.
You can look it up in the stores searching for 'SelectPicturesAvalonia'
There is an instructional video available [here](https://rosedeleyn.be/wp-content/uploads/SelectPicturesAvalonia.mp4)

**Reply by Yveske1966** [reply]
🫣 Pictures are not shown 🫣
A new version is in review with Microsoft and Apple
With my 'app'ologies
Yveske
